rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientProfileActivity.java

```diff
@@ -37,16 +38,17 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        RDTApplication.getInstance().setCurrentActivity(this);
```

Comment:
Move this to the register activity.

Reply:
done.

Reply:
Add unit tests to verify that if the current activity is patientregister, logout is not called.

Reply:
Also add a test to verify that patientregister sets itself as the currentactivity onresume and clears it onpause.

Reply:
test cases added